src/Installer/Database/PostgresDatabase.php

```diff
@@ -22,10 +22,11 @@ class PostgresDatabase extends AbstractDatabase
 
     public function getCreateDatabaseCommand()
     {
+        $pass     = !empty($this->pass) ? 'PGPASSWORD='.escapeshellarg($this->pass).' ' : '';
```

Comment:
I have only seen command specific environment variables set with `env`, EG: `env FOO=1 command` - would is be more correct to do that here?